src/account.js

```diff
@@ -3,7 +3,7 @@
  */
 
 const Client = require('./client')
-const API = require('./api')
+const API = require('./api').default
```

Comment:
a quirk of importing ts into js is that it exports as an object with a `default` property. this is how you can access the default export in a js import.

Reply:
this is a nice pr comment, I think it would be even nicer in this file for posterity